scripts/install

```diff
@@ -14,110 +8,96 @@ source /usr/share/yunohost/helpers
 #=================================================
 
 salt=$(ynh_string_random --length=40)
-ynh_app_setting_set --app=$app --key=salt --value=$salt
+ynh_app_setting_set --key=salt --value=$salt
 
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing dependencies..." --weight=1
+ynh_script_progression "Installing dependencies..."
 
-ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
+ynh_nodejs_install
```

Comment:
Why is the nodejs version removed?

Reply:
more context on helpers_v2.1: https://yunohost.org/en/packaging_apps_helpers_v2.1